### PR TITLE
fix: Stroke in bookmark icons

### PR DESCRIPTION
**Pull request by yagopv**
The stroke in the bookmark icons is too tiny when sm and vivid colors

### CODE REVIEW

**Review by DaniSomoza — APPROVED** [review]
Approved!

File: src/dataDisplay/Icon/images/bookmark.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default {
         <path
           className="icon-stroke"
```
Comment (katspaugh) [inline src/dataDisplay/Icon/images/bookmark.tsx:13]: Could you kill the classnames since you're at it? :)

Comment (yagopv) [reply src/dataDisplay/Icon/images/bookmark.tsx:13]: do you mean adding the fill directly in the svgs?

Comment (katspaugh) [reply src/dataDisplay/Icon/images/bookmark.tsx:13]: Oh, I thought this class doesn't do anything. If it sets the color globally, please ignore my comment.

**Pull request merged**